apps/evans/lib/dates.ts

```diff
@@ -40,3 +40,16 @@ export function spanishShortMonth(short: string): string {
   }
   return short;
 }
+
+export function formatDuration(seconds: number): string {
```

Comment:
`lib/dates.ts` seems like a weird place for this file...

also, does it handle edge cases well? can you make a few tests showing that < 60 seconds formats well, and over one hour as well? i've been burned by those sorts of things when i've written formatters like this in the past.

Reply:
I thought it has to do with durations which are sort of times which are sort of dates, but yeah that might be a stretch. Do you want it somewhere else?